.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -16,22 +16,27 @@ body:
         - The feature or configuration option you encounter the issue with.
         - The expected behavior.
         - The actual behavior (and how it deviates from the expected behavior, if it is not immediately obvious).
-        - Relevant Ghostty logs or other stacktraces.
-        - Relevant screenshots, screen recordings, or other supporting media (as needed).
+        - Screenshots, screen recordings, or other supporting media (as needed).
         - If this is a regression of an existing issue that was closed or resolved, please include the previous item reference (Discussion, Issue, PR, commit) in your description.
 
         >[!TIP]
         > **Not sure what information to include?**
         > Here are some recommendations:
-        > - **Input issues:** include your keyboard layout, a screenshot of the terminal inspector's logged keystrokes (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
+        > - **Input issues:** include your keyboard layout, a screenshot of the terminal inspector's logged keystrokes from the Terminal Inspector's "Keyboard" tab (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
         > - **Font issues:** include the problematic character(s), the output of `ghostty +show-face` for these character(s), and if they work in other applications.
         > - **VT issues (including image rendering issues):** attach an [asciinema](https://docs.asciinema.org/getting-started/) cast file, shell script, or text file for reproduction.
```

@pluiedev's suggestion isn't too bad and is definitely better than “VT”.
```suggestion
        > - **Terminal emulation issues (including image rendering issues):** attach an [asciinema](https://docs.asciinema.org/getting-started/) cast file, shell script, or text file for reproduction.
```
(If you accept this, co-author Leah instead of me.)

---

GitHub only allows a co-author through the UI from the person who suggests the changes.  I named her in the message and description since I could not be bothered to try and manually configure a co-author

---

Lol if you're fine with force pushing,
```
Co-authored-by: Leah Amelia Chen <hi@pluie.me>
```
Would work.